src/FixedDataTableCellGroup.js

```diff
@@ -82,7 +82,7 @@ var FixedDataTableCellGroupImpl = React.createClass({
       if (!recycable || (
             currentPosition - props.left <= props.width &&
             currentPosition - props.left + columnProps.width >= 0)) {
-        var key = 'cell_' + i;
+        var key = columnProps.columnKey || 'cell_' + i;
```

Comment:
@wcjordan There was discussion of adding a unique check for column keys. Do we still want to include that?